src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -112,7 +112,7 @@ internal void Execute(Message message, string sessionId)
             else if (message is GetLibraryItemsMessage)
             {
                 OnResultReady(this, new ResultReadyEventArgs(
-                    new LibraryItemsListResponse(dynamoModel.SearchModel.GetAllLibraryItemsByCategory()),
+                    new LibraryItemsListResponse(dynamoModel.SearchModel.GetAllLibraryItemsByCategory(dynamoModel)),
```

Comment:
now `SearchModel` doesn't know about dynamoModel but it's needed for getting all necessary information of each node